docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.7"
+services:
+  elastic:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
+    container_name: elastic
+    environment:
+      - node.name=elastic
+      - discovery.type=single-node
+      - cluster.name=docker-cluster
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - elastic:/usr/share/elasticsearch/data
+    ports:
+      - 9200:9200
+    networks:
+      - backend
+    restart: on-failure
+
+  natlas-server:
+    image: natlas-server
+    ports:
+      - 5000:5000
+    networks:
+      - backend
+    environment:
+      - ELASTICSEARCH_URL=http://elastic:9200
+      - BIND_ADDRESS=0.0.0.0
+      - NATLAS_SCAN_LOCAL=False
```

Comment:
NATLAS_SCAN_LOCAL is an agent configuration, not a server configuration. Also, NATLAS_SCAN_LOCAL's default value is False. https://github.com/natlas/natlas/tree/main/natlas-agent#the-config

Reply:
ugh. That was moved when I was restructuring it a bit. Moving back down